g11n-ws/modules/md-data-api-s3impl/src/main/java/com/vmware/vip/messages/data/dao/impl/S3MultComponentDaoImpl.java

```diff
@@ -40,7 +40,11 @@
       }
       for (String component : components) {
          for (String locale : locales) {
-            bundles.add(oneComponentDao.get2JsonStr(productName, version, component, locale));
+        	 try {
+               bundles.add(oneComponentDao.get2JsonStr(productName, version, component, locale));
+        	 }catch(DataException e ) {
+        		 logger.warn(e.getMessage(), e);
```

Comment:
Should this be error level?

Reply:
because  some components or locales can't get translations is warning case when get mult component translations